cpu/native/startup.c

```diff
@@ -250,6 +253,19 @@ void usage_exit(int status)
     real_exit(status);
 }
 
+#ifdef __APPLE__
+/* Taken from the sources of Apple's dyld launcher
+ * https://github.com/opensource-apple/dyld/blob/3f928f32597888c5eac6003b9199d972d49857b5/src/dyldInitialization.cpp#L85-L104
```

Comment:
Yes, I think it's better to have a specific commit in this case since we are referring to some lines which could move around as master evolves.